jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -460,18 +460,14 @@ bool java_bytecode_parsert::parse()
 #define ACC_ANNOTATION   0x2000
 #define ACC_ENUM         0x4000
 
-#ifdef _MSC_VER
-#define UNUSED
-#else
-#define UNUSED __attribute__((unused))
-#endif
+#define UNUSED_u2(x) { const u2 x = read_u2(); (void)x; } (void)0
```

Comment:
Why the `(void)0` at the end?

Reply:
Just to make sure uses of the macro can (and do) have a semicolon at the end. Without the `(void)0` doing so would likely yield a warning (as it would expand to a semicolon after a closing brace).